### DISCUSSION TITLE
Set PYTHONUNBUFFERED env variable

### DISCUSSION THREAD

**Discussion by fkromer** [tiangolo/uwsgi-nginx-flask-docker#375] [Questions] [answered]
What do you think about [preventing buffered stdout and stderr streams](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED) by adding `ENV PYTHONUNBUFFERED 1` to the dockerfiles?

**Comment by tiangolo** [comment, marked as answer]
Yep! I added that a while back to the parent Docker image. :nerd_face:
